### PR TITLE
Allow ideas get-extensions to run with macOS

### PR DESCRIPTION
**Pull request by adowling2**
## Changes proposed in this PR:
- Replaced exception with a printed warning

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

**Comment by adowling2** [comment]
@ksbeattie @eslickj This one line change allows me to test installing the macOS binaries. Please provide feedback if you prefer something instead of a print statement for the warning message. Let's wait to merge until we have the binaries tested for multiple people.

**Comment by eslickj** [comment]
I made PR #163, which is a more permanent update for this.  It also let's you skip the checksum check, which I think will be good for us to test the Mac binaries, since it's a little like a "I know it's not quite right but I want it anyway" option.

File: idaes/solvers.py [hunk 1]
```diff
@@ -100,7 +100,8 @@ def download_binaries(
         _log.debug("URLs \n  {}\n  {}\n  {}".format(url, solvers_from, libs_from))
         _log.debug("Destinations \n  {}\n  {}".format(solvers_tar, libs_tar))
         if platform == 'darwin':
-            raise Exception('Mac OSX currently unsupported')
+            # raise Exception('Mac OSX currently unsupported')
+            print("WARNING: macOS support is experimental.")
```
Comment (ksbeattie) [inline idaes/solvers.py:104]: I think a `_log.warn()` might be more appropriate here...

**Comment by eslickj** [comment]
I'll go ahead an close this.  Feature is in #163 now.

**Pull request closed without merging**